src/library/scala/collection/immutable/Set.scala

```diff
@@ -26,17 +26,17 @@ import parallel.immutable.ParSet
  *  @define coll immutable set
  */
 trait Set[A] extends Iterable[A]
-                with GenSet[A]
+//                with GenSet[A]
```

Comment:
It's not changing any of our errors, and this is how it was in the first
place. So, IMHO it's better to leave it.

On Thu, Nov 19, 2015 at 5:28 PM, Tiark Rompf <notifications@github.com>
wrote:

> In src/library/scala/collection/immutable/Set.scala
> <https://github.com/TiarkRompf/scala/pull/8#discussion_r45410772>:
>
> > @@ -26,17 +26,17 @@ import parallel.immutable.ParSet
> >   *  @define coll immutable set
> >   */
> >  trait Set[A] extends Iterable[A]
> > -                with GenSet[A]
> > +//                with GenSet[A]
>
> why is this necessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TiarkRompf/scala/pull/8/files#r45410772>.
>